### PR TITLE
Prepare 2.8.1 release

### PR DESCRIPTION
**Pull request by gzussa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
-## 2.9.0 (Unreleased)
+## 2.8.1 (Unreleased)
+
+IMPROVEMENTS:
+* `monitor`: Add monitor `force_delete` parameter ([#535](https://github.com/terraform-providers/terraform-provider-datadog/pull/535)) Thanks [@ykyr](https://github.com/ykyr)
+* `provider_test`: Allow to pass http client when creating test provider ([#541](https://github.com/terraform-providers/terraform-provider-datadog/pull/541)) Thanks [@txominpelu](https://github.com/txominpelu)
```
Comment (nmuesch) [inline CHANGELOG.md:5]: This change is only to a test file, should we remove it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+
+BUGFIXES:
+* `dashboard`: Safely access index field ([#536](https://github.com/terraform-providers/terraform-provider-datadog/pull/536))
+* `dashboard`: Set title and title_align properly on heatmap widget ([#539](https://github.com/terraform-providers/terraform-provider-datadog/pull/539))
+* `ip_ranges`: Fix data source for IPRanges ([#542](https://github.com/terraform-providers/terraform-provider-datadog/pull/542))
+* `monitor`: Fix indent in datadog_monitor ([#543](https://github.com/terraform-providers/terraform-provider-datadog/pull/543)) Thanks [@nekottyo](https://github.com/nekottyo)
```
Comment (nmuesch) [inline CHANGELOG.md:11]:
```suggestion
* `monitor`: Fix indent in datadog_monitor docs example ([#543](https://github.com/terraform-providers/terraform-provider-datadog/pull/543)) Thanks [@nekottyo](https://github.com/nekottyo)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,15 @@
-## 2.9.0 (Unreleased)
+## 2.8.1 (Unreleased)
+
+IMPROVEMENTS:
+* `monitor`: Add monitor `force_delete` parameter ([#535](https://github.com/terraform-providers/terraform-provider-datadog/pull/535)) Thanks [@ykyr](https://github.com/ykyr)
```
Comment (nmuesch) [inline CHANGELOG.md:4]: This applies to all, but should we use the full resource name for consistency, e.g. `datadog_monitor`

File: CHANGELOG.md (hunk 1 above)
Comment (gzussa) [inline CHANGELOG.md:5]:
```suggestion
```

File: CHANGELOG.md (hunk 3 above)
Comment (gzussa) [inline CHANGELOG.md:4]:
```suggestion
* `datadog_monitor`: Add monitor `force_delete` parameter ([#535](https://github.com/terraform-providers/terraform-provider-datadog/pull/535)) Thanks [@ykyr](https://github.com/ykyr)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,15 @@
...
+* `provider_test`: Allow to pass http client when creating test provider ([#541](https://github.com/terraform-providers/terraform-provider-datadog/pull/541)) Thanks [@txominpelu](https://github.com/txominpelu)
+
+BUGFIXES:
+* `dashboard`: Safely access index field ([#536](https://github.com/terraform-providers/terraform-provider-datadog/pull/536))
```
Comment (gzussa) [inline CHANGELOG.md:8]:
```suggestion
* `datadog_dashboard`: Safely access index field ([#536](https://github.com/terraform-providers/terraform-provider-datadog/pull/536))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,15 @@
...
+* `dashboard`: Set title and title_align properly on heatmap widget ([#539](https://github.com/terraform-providers/terraform-provider-datadog/pull/539))
```
Comment (gzussa) [inline CHANGELOG.md:9]:
```suggestion
* `datadog_dashboard`: Set title and title_align properly on heatmap widget ([#539](https://github.com/terraform-providers/terraform-provider-datadog/pull/539))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,15 @@
...
+* `ip_ranges`: Fix data source for IPRanges ([#542](https://github.com/terraform-providers/terraform-provider-datadog/pull/542))
```
Comment (gzussa) [inline CHANGELOG.md:10]:
```suggestion
* `datadog_ip_ranges`: Fix data source for IPRanges ([#542](https://github.com/terraform-providers/terraform-provider-datadog/pull/542))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,15 @@
...
+* `monitor`: Fix indent in datadog_monitor docs example ([#543](https://github.com/terraform-providers/terraform-provider-datadog/pull/543)) Thanks [@nekottyo](https://github.com/nekottyo)
```
Comment (gzussa) [inline CHANGELOG.md:11]:
```suggestion
* `datadog_monitor`: Fix indent in datadog_monitor docs example ([#543](https://github.com/terraform-providers/terraform-provider-datadog/pull/543)) Thanks [@nekottyo](https://github.com/nekottyo)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,14 @@
...
+
+IMPROVEMENTS:
+* `datadog_monitor`: Add monitor `force_delete` parameter ([#535](https://github.com/terraform-providers/terraform-provider-datadog/pull/535)) Thanks [@ykyr](https://github.com/ykyr)
+
+BUGFIXES:
+* `datadog_dashboard`: Safely access index field ([#536](https://github.com/terraform-providers/terraform-provider-datadog/pull/536))
+* `datadog_dashboard`: Set title and title_align properly on heatmap widget ([#539](https://github.com/terraform-providers/terraform-provider-datadog/pull/539))
+* `datadog_ip_ranges`: Fix data source for IPRanges ([#542](https://github.com/terraform-providers/terraform-provider-datadog/pull/542))
+* `datadog_monitor`: Fix indent in datadog_monitor docs example ([#543](https://github.com/terraform-providers/terraform-provider-datadog/pull/543)) Thanks [@nekottyo](https://github.com/nekottyo)
+
```
Comment (zippolyte) [inline CHANGELOG.md:11]: Should we have an entry about the fix to synthetics device ids ?

**Pull request merged**